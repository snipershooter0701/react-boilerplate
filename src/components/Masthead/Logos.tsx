import React from 'react';
import styled from 'styled-components/macro';
import { ReactComponent as CRALogo } from './cra-logo.svg';
import { ReactComponent as RPLogo } from './rp-logo.svg';
import { ReactComponent as PlusSign } from './plus-sign.svg';

export default function Logos() {
  return (
    <Wrapper>
      <CRALogo className="logo" />
      <PlusSign className="sign" />
      <RPLogo className="logo" />
    </Wrapper>
  );
}

const Wrapper = styled.div`
  display: flex;
  align-items: center;

  .logo {
    width: 4.5rem;
    height: 4.5rem;
  }

  .sign {
    width: 2rem;
    height: 2rem;
    margin: 0 2rem;
  }
`;
