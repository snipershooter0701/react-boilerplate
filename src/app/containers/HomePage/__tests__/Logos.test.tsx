import React from 'react';
import renderer from 'react-test-renderer';
import { Logos } from '../Logos';

describe('<Logos />', () => {
  it('should render and match the snapshot', () => {
    const renderedOutput = renderer.create(<Logos />).toJSON();
    expect(renderedOutput).toMatchSnapshot();
  });
});
