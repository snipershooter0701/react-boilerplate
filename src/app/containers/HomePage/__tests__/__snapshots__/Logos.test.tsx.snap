// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logos /> should render and match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 .logo {
  width: 4.5rem;
  height: 4.5rem;
}

.c0 .sign {
  width: 2rem;
  height: 2rem;
  margin: 0 2rem;
}

<div
  className="c0"
>
  <IMAGE_MOCK
    className="logo"
  />
  <IMAGE_MOCK
    className="sign"
  />
  <IMAGE_MOCK
    className="logo"
  />
</div>
`;
